<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>sForm</class>
 <widget class="QWidget" name="sForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>141</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>380</width>
    <height>141</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Vare/Sags nr. eller Fri tekst søgning</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="SeekT" >
         <property name="text" >
          <string>Søg i Tilbud</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SeekO" >
         <property name="text" >
          <string>Søg i Ordre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SeekK" >
         <property name="text" >
          <string>Søg i Konsignation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SeekA" >
         <property name="text" >
          <string>Søg i Arbejdskort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SeekTxt" >
         <property name="text" >
          <string>Søg i Tekst</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11" >
             <property name="text" >
              <string>Sorter i</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cust_order" >
             <property name="text" >
              <string>Kunde nr. orden</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="lines_cust_order" >
             <property name="text" >
              <string>Vare per Kunde</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="date_order" >
             <property name="text" >
              <string>Dato orden</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>5</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>I Perioden</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Fra</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="p_From" >
               <property name="minimumDate" >
                <date>
                 <year>2005</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="displayFormat" >
                <string>dd-MMM-yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Til</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="p_To" >
               <property name="date" >
                <date>
                 <year>2005</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="minimumDate" >
                <date>
                 <year>2005</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="displayFormat" >
                <string>dd-MMM-yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="seekTxtInput" />
         </item>
         <item>
          <widget class="QPushButton" name="nr_txtSeek" >
           <property name="text" >
            <string>Søg</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
